build.psm1

```diff
@@ -1571,6 +1571,8 @@ function Start-PSxUnit {
 
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-linux`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```